cylc/flow/option_parsers.py

```diff
@@ -209,11 +209,15 @@ def add_std_options(self):
 
         if self.icp:
             self.add_option(
-                "--icp",
+                "--initial-cycle-point", "--initial-point", "--icp", "--ict",
```

Comment:
Why `ict`? Is it another way to abbreviate initial cycle point?

Reply:
`ict`  (initial cycle time) is the Cylc-5 way of calling `icp` (initial cycle point). We can probably remove it.

Reply:
Aaahh, that makes sense. All good if stay too, was more curiosity. Thanks!